app/assets/javascripts/Components/Result/url_viewer.jsx

```diff
@@ -5,15 +5,16 @@ export class URLViewer extends React.Component {
     super(props);
     this.state = {
       url: "",
+      embeddedURL: "",
     };
   }
 
   componentDidMount() {
     this.configDisplay();
   }
 
-  componentDidUpdate(prevProps) {
-    if (prevProps.externalUrl !== this.props.externalUrl) {
+  componentDidUpdate(prevProps, prevState) {
+    if (prevProps !== this.props) {
```

Comment:
Javascript objects are not equal as long as they don't refer to the same object in memory. This means that this if statement may be true even if the contents of `prevProps` and `this.props` are identical. If you're doing this comparison, it is safer to compare the content of the props objects, not the objects themselves.